Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -272,12 +273,12 @@ - (void)viewDidLoad {
                            @"Fitness Task",
                            @"GAIT Task",
                            @"Memory Game Task",
-                           @"PSAT Task",
-                           @"Reaction Time Task",
-                           @"Timed Walk Task",
+                           @"Two Finger Tapping Task",
                            @"Tone Audiometry Task",
+                           @"Reaction Time Task",
                            @"Tower Of Hanoi Task",
-                           @"Two Finger Tapping Task",
+                           @"PSAT Task",
+                           @"Timed Walk Task",
```

Comment:
@YuanZhu-apple you're right, my bad. I added the Hole Peg Test task and also fixed an issue with the method name.